api/grpc/Server.go

```diff
@@ -131,7 +131,7 @@ func NewServer(h application.AccessAppService, l application.LicenseAppService,
 func (s *Server) Serve(wait *sync.WaitGroup) error {
 	defer wait.Done()
 
-	ls, err := net.Listen("tcp", ":"+s.ServerConfig.GrpcPort)
+	ls, err := net.Listen("tcp", "localhost:"+s.ServerConfig.GrpcPort)
```

Comment:
Unfortunately, I don't think changing to localhost is going to work after all. The default binding address of `0.0.0.0` makes the port accessible to kube services, but binding to `localhost/127.0.0.1` does not. Still looking for a good source (as opposed to some bad ones I found on google), but this makes sense.

Reply:
reverted it back to how it was before :)